### ISSUE TITLE
serCustom method not run outside database

### ISSUE THREAD

**Issue by smallik** [orientechnologies/orientdb#5700]
Version: orientdb-community-2.1.9

bug description: when using the graph database. ie schema changes are automatically run outside the current transaction. But in the setCustom method (both on vertex as well as property) that is not the case. it is throwing exception saying you cant make schema changes during a transaction.
The rest of the graph schema changes like creating class, adding property does not have a problem as it is automatically run outside a transaction.

To recreate:
```
OrientGraphFactory factory = new OrientGraphFactory("plocal:"+, , ).setupPool(1, 1);
graph=factory.getTx();
OrientVertexType typedef=graph.getVertexType(param.File.name());
prop = def.createProperty("attribute name", OType.LINK, OType.STRING);
prop.setCustom("someparam","param value");
```

Workaround:
```
prop=graph.executeOutsideTx(new OCallable<OProperty, OrientBaseGraph>() {
     @Override
     public OProperty call(final OrientBaseGraph g) {
          OProperty prop=def.getProperty("attrname");
          return prop.setCustom("someparam","param value");
     }
}, "");
```

**Comment by smallik** [comment]
Awesome, that was quick!

**Comment by luigidellaquila** [comment]
Hi @smallik 

the fix is easy, but it does not only involve setCustom(), I could reproduce the same problem with setMax(), setMin() and all the other constraints

I'm working to a fix

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @smallik 

I pushed a fix on 2.1.x branch, together with some test cases.
The fix will be released next week with 2.1.11

I'm closing this issue, please reopen in case you see any problems

Thanks

Luigi
